woocommerce/woocommerce
# Checkout block: Fix issue with vertical alignment in `Terms and Conditions` innerblock checkbox

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





- Fixed the alignment issue between the checkbox and the associated text in the WooCommerce checkout block `Terms and Conditions`.
- The issue was caused by a missing `wc-block-components-checkbox__label` class on the <span> element wrapping the terms and conditions label.
- The fix adds the missing class to the span element, ensuring the label text is properly aligned with the checkbox.

Closes #52722



### How to test the changes in this Pull Request:



1. Navigate to the WooCommerce checkout page.
2. Verify that the "Terms and Conditions" checkbox and its label text are now vertically centered and aligned.
3. Check that no other elements or styles are broken as a result of this fix.
4. Ensure the page layout is consistent across different screen sizes.
5. Test in both regular (Twenty Twenty Four, Storefront) and custom themes to confirm no conflicts arise.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Fix incorrect alignment of T&C checkbox and label

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
